### ISSUE TITLE
Substance and Deadline

### ISSUE THREAD

**Issue by FredP** [asztalosdani/deadline-substance-painter-plugin#2]
Looking for ways to offload texture baking to Deadline from Substance.  Is that what this plugin config does?

**Comment by asztalosdani** [comment]
Yes, exactly. You need this repo for Deadline and the other repo for Substance. https://github.com/asztalosdani/substance-painter-to-deadline

**Comment by FredP** [comment]
Does this work with the latest version of Substance from Adobe?

As far as installation, I assume the plugin goes into the 
Substance/plugins/ folder.  But for Deadline, do I just add this into 
the deadlinerepo/plugins/ folder?  I did this, and Deadline shows that 
there is a substance plugin, but that there are no parameters defined in 
the plugin window.

Thanks for your help.
Fred

**Comment by asztalosdani** [comment]
>Does this work with the latest version of Substance from Adobe?

Unfortunately it's been a while since I worked with this, so I don't know.

> As far as installation, I assume the plugin goes into the Substance/plugins/ folder. 

I think yes.

>But for Deadline, do I just add this into the deadlinerepo/plugins/ folder?

You have to copy (or clone) the contents of this repo to a `SubstancePainter` folder under `[Repository]/custom/plugins/` So `[Repository]/custom/plugins/SubstancePainter` is the full path

**Comment by FredP** [comment]
Understood.  Do you have any intentions or desire to look into updating 
this script if Adobe has caused issues with its functionality?

**Comment by asztalosdani** [comment]
I no longer have access to DL or SP, so I cannot update it confidently, but I am happy to review PRs.
